src/database.py

```
@@ -55,8 +66,12 @@ def insert_nutrition_data(self, nutrition_data: Dict[str, Any]) -> bool:
         영양 정보 데이터를 Supabase에 삽입
         """
         try:
-            result = self.client.table('Nutrition').insert(nutrition_data).execute()
-            logger.info(f"Nutrition data inserted successfully for product_id: {nutrition_data.get('product_id')}")
+            # 데이터 직렬화
+            serialized_data = self._serialize_data(nutrition_data)
+            result = self.client.table("Nutrition").insert(serialized_data).execute()
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)